persistence/src/commonMain/kotlin/com/wire/kalium/persistence/dao/ConversationDAOImpl.kt

```diff
@@ -22,8 +23,11 @@ class MemberMapper {
     }
 }
 
-class ConversationDAOImpl(private val conversationQueries: ConverationsQueries,
-                          private val memberQueries: MembersQueries): ConversationDAO {
+class ConversationDAOImpl(
+    private val conversationQueries: ConverationsQueries,
```

Comment:
Late to the party but maybe to keep in mind for next PR 👓 🙊 
```suggestion
    private val conversationQueries: ConversationsQueries,
```

Reply:
Yeaaaaaaaaah hahahah

It's an old typo that went through review and is being corrected little by little 😆